fix(time-picker): fix handlechange throw error when range input change

shlroland opened the pull request:
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

zazzaz committed:
chore: update changelog

zazzaz committed:
1.0.1-beta.7

shlroland committed:
fix(time-picker): fix handlechange throw error when range input change

shlroland committed:
fix(time-picker): fix handlechange throw error when range input change

JeremyWuuuuu commented:
Can you add some tests against this code? In case this gets broken in the future.

JeremyWuuuuu commented:
The change seems reasonable to me.

jw-foss committed:
fix(dialog): fix dialog bouncing (#955)

- Change rendered before open dispatches to avoid bouncing

jw-foss committed:
fix(collapse): fix collapse not emitting change (#949)

- Add emit when active changes

Ryan2128 committed:
fix(menu): fix open method error (#952)

* fix(menu): fix open method error

* test(menu): add open method error test case

zazzaz committed:
fix: export locale (#950)

kooriookami committed:
fix(popover): popover with pure should has border (#954)

fix #942

zouhangwithsweet committed:
fix: fix some components import type (#959)

kooriookami committed:
fix(docs): lang="sass" to lang="scss" (#970)

justwiner committed:
fix(table): fix the style bug when the parent container is display: flex (#969)

Fix the bug of abnormal height and width of table when the parent container is display: flex

fix #922

Co-authored-by: winerlu <winerlu@tencent.com>

Commit:
fix(docs): change upload docs #966 (#976)

zazzaz committed:
chore: update changelog

zazzaz committed:
1.0.1-beta.8

wujieZ committed:
style: arrow function (#982)

kooriookami committed:
fix(scrollbar): native scrollbar not hide when `native` is not true (#971)

negative value of margin-bottom not work in certain scenarios.

csvwolf committed:
feat(docs): import EP in demobox to support Composition API docs (#984)

JeremyWuuuuu commented:
Any updates? Or I am going to close this PR @shlroland

jw-foss committed:
feat(virtual-list): first implementation of virtual list (#790)

jw-foss committed:
fix(dialog): fix/dialog-header-slot (#997)

- Rename the slot name from 'header' to title

kooriookami committed:
fix(table): fix table filter-panel style (#943)

Liiked committed:
feat(menu): update menu spec (#980)

* feat(menu): update menu spec

* feat(menu): update to rAF

* feat(menu): remove raf method

Co-authored-by: liik <linyunqianpp@126.com>

Ryan2128 committed:
fix(tree): fix keyboard navigation bug (#995)

* fix(tree): fix keyboard navigation bug

* test(tree): add navigate test case

SimonaliaChen committed:
style: fix typo (#1008)

Ther-su committed:
fix dialog can't scroll if the content too large (#993)

* fix(dialog): fix dialog can't scroll if the content too large

fix dialog can't scroll if the content too large

fix #991

* fix(dialog): make the mask can scroll

make the mask can scroll

fix #991

shlroland committed:
fix(date-picker): fix range-picker

when calEndDate is before minDate, the start and the end cell cannt be updated

shlroland committed:
Merge branch 'fix/time-picker' of github.com:shlroland/element-plus into fix/time-picker

JeremyWuuuuu commented:
.. Please use rebase instead of merge on top of your branch. That contaminates the commit history.

JeremyWuuuuu commented:
Closing due to stale.
